### DISCUSSION TITLE
How do I set max age on _nuxt generated .mjs files

### DISCUSSION THREAD

**Discussion by Jlaird** [nuxt/nuxt#16650] [Questions]
I am trying to get the CDN to cache the .mjs and .css files. Currently Nuxt doesn't seem to set a cache control header. Is it posable through the nuxt config to set this?

**Comment by TheAlexLichter** [comment]
Related issue: https://github.com/nuxt/framework/issues/7287

**Comment by Jlaird** [comment]
@manniL Why do you think that is the same? That ticket is talking about assets to the website. I am talking about the Vite build files in _nuxt. They used to be set to 2 days immutable pre rc.01. Now the cache control header is not set at all. My guess is that someone added a feature so it can be set in the config like nuxt 2.

If it helps this is how we set them in vue 2.
``` 
 render: {
    static: {
      maxAge: 2592000000, // Cache for 30 days
    },
  },
